OWNERS_ALIASES

```diff
@@ -0,0 +1,17 @@
+# See the OWNERS docs: https://git.k8s.io/community/docs/devel/owners.md
+
+aliases:
+  sig-cloud-provider-leads:
```

Comment:
So this is something that has been bothering me for a long time across several projects... Why is that alias defined here at all? Shouldn't it be coming from kubernetes/org, aka [here](https://github.com/kubernetes/org/blob/master/OWNERS_ALIASES#L38).

I believe the Prow plugin walks all these redefinitions, but I think for top-level teams, the canonical source should be the `org` repo, otherwise this is a maintenance nightmare.

Reply:
I personally think the only use of `OWNER_ALIASES` for our projects would be define a group of our choosing for reviewers vs approvers, but at that point, we should just list them directly in `OWNERS`. Either way you have to modify a file. I see no use for `OWNER_ALIASES`. The `cloud-provider-leads` is a great example -- the list of people there are about to change, but this file here would override the top-level one, giving someone approval rights to our repo that is no longer a co-chair of the SIG.

/ rant

Reply:
fair point, I wasn't aware that prow could walk the definitions back to `kubernetes/org`, I'll amend the file 👍 

I still see the value in keeping the `OWNERS_ALIASES` file as it defines who is part of what "local" group while `OWNERS` specify the groups that are able to approve and/or review.

Reply:
Cool. To be clear, I *think* that's the case, but haven't definitely proven it. I've looked through the Prow plugin code around this before, and think that's happening, but I wanted to start the conversation. I should probably just ask the test-infra folks sometime.

Reply:
:wave: Just want to confirm that prow will _not_ walk back definitions to k/org or any other repo. There is an open issue for this: https://github.com/kubernetes/test-infra/issues/10065, but it's not possible today. Members for all aliases need to be explicitly defined in the OWNERS_ALIASES file.

Reply:
Ok, added it back 😄, thanks @nikhita and @codenrhoden for helping in getting to the bottom of this, it was indeed confusing 😅